src/windows/glog/logging.h

```diff
@@ -1099,16 +1099,16 @@ const LogSeverity GLOG_0 = GLOG_ERROR;
 
 // Log only in verbose mode.
 
-#define VLOG(verboselevel) LOG_IF(INFO, VLOG_IS_ON(verboselevel))
+#define VLOG(verboselevel) LOG_IF(INFO, VLOG_IS_ON(verboselevel)) << "V" << verboselevel << ": "
```

Comment:
Would this `V: ` come after all this:
`[m-1] I1011 19:23:12.825950 61469 memtable_list.cc:392]`

That seems like it could be ignored, could we somehow replace I1011 with V1011?

Reply:
That will be quite harder to do. And We want something like V1, V2, V3 for the verbosity level right?